src/gatsby-node.js

```diff
@@ -20,7 +20,7 @@ export const onCreateWebpackConfig = (
 };
 
 /**
- * Moves all js and css files into timestamp-named folder
+ * Moves assets to assets folder
```

Comment:
So, I have two questions about your suggestion:
- In what way is "a custom folder" better than the "assets folder"? I mean, it IS the assets folder that we are moving the assets to: it is declared by the  `assetsPrefix` in your Gatsby config.
- Why include suggestions for what people could name their assets folder to here? For what I can see, it has nothing to do with the implementation of this method, but maybe I'm missing something here?

Reply:
Hmm, i'm not totally sold on it either, but it's a pointer to aid someone who is digging through the abstractions of third party code. It's a direct reference to `"custom_asset_folder"` in the readme.

Reply:
I now see that this method is taking `pathPrefix` as it's argument. I tried to run the plugin with the following `gatsby-config.js`:
```
  assetPrefix: 'assets',
  pathPrefix: 'blog',
```
and with that configuration, the argument has this value: `/assets/blog`. So what this plugin is actually doing right now is to move the files to a directory in `public` that is named `assetPrefix`/`pathPrefix/`. That is probably not what you would like... Is it possible to replace `pathPrefix` with the actual `assetPrefix`?

Reply:
Also, about your comment above, I think it would be better to point someone that is digging to the actual Gatsby configuration rather than something mentioned in this plugin's readme.

Reply:
@raurir any thoughts here?